lib/ecom/session.ts

```diff
@@ -0,0 +1,42 @@
+import { createCookieSessionStorage } from '@remix-run/node';
+import { Tokens } from '@wix/sdk';
+import { getWixClientId, initializeEcomApi } from './api';
+
+type SessionData = {
+    wixEcomTokens: Tokens;
+    wixClientId: string;
+};
+
+const { getSession, commitSession } = createCookieSessionStorage<SessionData, void>({
+    cookie: {
+        name: '__session',
+        maxAge: 3600 * 24 * 30 * 3, // 3 months
```

Comment:
only if containing February :) in non-leap year

Reply:
> only if containing February :) in non-leap year
will update to 90 days

> why 3 months?
just my imagination. do you have some more meaningful period?

Reply:
100 is a round number

Reply:
maybe we can use the lifespan of wix tokens?

Reply:
changed to 100 days.
@yurii-ve ,  we can not use wix tokens at the momen of session initialization, cause this is kinda 'static' session cookie initialization, we have no wix tokens yet